examples/mplot3d/box3d.py

```diff
@@ -0,0 +1,97 @@
+"""
+=================================
+3D surface plot using Matplotlib
+=================================
+by: Iury T. Simoes-Sousa (iuryt)
+
+The strategy is to select the data from each surface and plot
+contours separately.
+To use this feature you need to have gridded coordinates.
+
+The contour plot from Matplotlib has zdir argument that defines
+the normal coordinate to the plotted surface.
+
+The offset argument defines the offset applied to the contourf surface.
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+# Define dimensions
+Nx, Ny, Nz = 100, 300, 500
+X, Y, Z = np.meshgrid(np.arange(Nx), np.arange(Ny), -np.arange(Nz))
+
+# Create fake data
+da = (((X+100)**2 + (Y-20)**2 + 2*Z)/1000+1)
+
+vmin = da.min()
+vmax = da.max()
+# Key arguments for contour plots
+kw = {
+    'vmin': vmin,
+    'vmax': vmax,
+    'levels': np.linspace(vmin, vmax, 10),
+    'cmap': 'viridis',
+}
```

Comment:
It's a good idea to try and keep examples compact. Ideally, they fit on one screen, which helps readability a lot.

```suggestion
kw = {
    'vmin': data.min(),
    'vmax': data.max(),
    'levels': np.linspace(data.min(), data.max(), 10),
}
```
And move this directly before the `contourf()` calls.

`cmap': 'viridis'` is not necessary as it's the default.